host-contracts/contracts/InputVerifier.sol

```
@@ -326,34 +368,85 @@ contract InputVerifier is UUPSUpgradeableEmptyProxy, Ownable2StepUpgradeable, EI
             }
         }
     }
+    */
 
     /**
-     * @notice          Returns the list of signers.
+     * @notice          Returns the list of signers of a specific context ID.
      * @dev             If there are too many signers, it could be out-of-gas.
+     * @param contextId The context ID of the signer addresses to return.
      * @return signers  List of signers.
      */
-    function getCoprocessorSigners() public view virtual returns (address[] memory) {
+    function getCoprocessorSigners(uint256 contextId) public view virtual returns (address[] memory) {
```

can you make this function maybe also return a boolean `isSuspended` ? not just the list of signers. Could be helpful for debugging.

---

are you sure about this ? I feel like we should keep this function simple. We could provide another function for getting the status for debugging if really needed

---

Sorry `isSuspended` was wrong. I think we should return actually `isActiveOrSuspended` here to know if the input would revert or not.
If `isActiveOrSuspended` is `true` then the relayer-sdk signature check will pass, otherwise, it should raise an error.